### PR TITLE
PPTP-1943 - added content to the organisation-already-registered page

### PR DESCRIPTION
**Pull request by mauro-fanton**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [x] Required Environment Config has been amended/added 
 - [ ] Links to dependencies have been included (BE/FE work) - N/A
 - [x] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave - N/A

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -464,6 +464,9 @@ soleTraderEntityVerification.failure.detail = Check your Self Assessment records
 
 duplicateSubscription.title = Your organisation is already registered
 duplicateSubscription.detail = Someone in {0} has already registered for Plastic Packaging Tax (PPT).
+duplicateSubscription.account.detail.link = [Go to your PPT account to]:
```
Comment (C-Thomp) [inline conf/messages.en:467]: [...] should be left out, they're to tell you it is a link in the SOT

**Review by C-Thomp — CHANGES_REQUESTED** [review]
Just those brackets in messages file and everything else looks good 👍

**Review by C-Thomp — APPROVED** [review]
✅

**Comment by C-Thomp** [comment]
Notes from Mauro
-Was not able to run the acceptance test locally as they are broken.
-Wont merge this PR until the acceptance test are fixed

File: conf/messages.en [hunk 2]
```diff
@@ -464,6 +464,9 @@ soleTraderEntityVerification.failure.detail = Check your Self Assessment records
 
 duplicateSubscription.title = Your organisation is already registered
 duplicateSubscription.detail = Someone in {0} has already registered for Plastic Packaging Tax (PPT).
+duplicateSubscription.account.detail.link = Go to your PPT account to:
```
Comment (mauro-fanton) [inline conf/messages.en:467]: Hi Carl, I am not sure if you have seen my previous comments, but I asked Lisa about the brackets and she said to leave them in. But fell free to ask her agin as she might misunderstood me.

File: conf/messages.cy [hunk 3]
```diff
@@ -462,6 +462,9 @@ soleTraderEntityVerification.failure.detail  = Gwiriwch eich cofnodion Hunanases
 
 duplicateSubscription.title  = Mae’ch sefydliad eisoes wedi’i gofrestru
 duplicateSubscription.detail  = Mae rhywun yn {0} eisoes wedi cofrestru ar gyfer y Dreth Deunydd Pacio Plastig.
+duplicateSubscription.account.detail.link = [Ewch i’ch cyfrif Treth Deunydd Pacio Plastig i wneud y canlynol]:
```
Comment (mauro-fanton) [inline conf/messages.cy:465]: Hi Carl, I am not sure if you have seen my previous comments, but I asked Lisa about the brackets and she said to leave them in. But fell free to ask her agin as she might misunderstood me.

File: conf/messages.en (hunk 2 above)
Comment (C-Thomp) [reply conf/messages.en]: Ah I hadn't. Changed back 👍

**Review by C-Thomp — APPROVED** [review]

**Pull request merged**